package.json

```diff
@@ -16,34 +15,47 @@
     "license": "ISC",
     "scripts": {
         "serve": "vue-cli-service serve",
-        "build": "vue-cli-service build",
-        "build-lib": "vue-cli-service build --target lib --name form-checkbox ./src/components/FormCheckbox.vue",
-        "lint": "vue-cli-service lint --fix"
+        "build": "npm run build:es & vue-cli-service build --target lib --name form-checkbox ./src/index.js",
```

Comment:
dopis do readme kt prikaz co robi a co mam spustat pred publish `build` ci `build:es` ?

Reply:
hmm, to akoze budeme mat v readmecku ako publishnut package? (je to inak npm run build)